pychromecast/controllers/media.py

```diff
@@ -67,6 +69,15 @@ def __init__(self):
         self.media_metadata = {}
         self.subtitle_tracks = {}
         self.current_subtitle_tracks = []
+        self.last_updated = time.time()
+
+    def get_current_time(self):
+        """ Get current seek time of video """
```

Comment:
1. It's not always a "video", it can also be audio; better change it to `Get current seek time of media`.
2. The method name is not very nice - `get_current_time` doesn't suggest seek times. Maybe `get_seek_time`.
3. Please provide a better docstring; i know this library is not too great with documentation, but still it would be good to specify what this method returns - the time *in seconds* since start of media/seek time.
4. What should this method return if nothing is playing? I think we should return `None` or something like that to indicate invalid call then.